mdd.py

```diff
@@ -230,20 +230,33 @@ def get_install_date():
 def get_system_info():
     logging.info("...get system info")
 
+    aarch64 = platform.machine() == "aarch64"
+
     def get_dmi(file_name: str):
         return get_command_output("cat /sys/devices/virtual/dmi/id/" + file_name)
 
+    if not aarch64:
```

Comment:
Better to `if aarch64:`, then handle the special ARM case. Happy path left.

Reply:
hmm - the current prioritize x86 - if I invert it prioritize aarch64 - which one is better :shrug: 

I will invert as per request.